types/decimal.go

```diff
@@ -33,6 +34,13 @@ var (
 	tenInt               = big.NewInt(10)
 )
 
+// Decimal errors
+var (
+	ErrEmptyDecimalStr      = errors.New("decimal string cannot be empty")
```

Comment:
shouldn't these be moved to the `sdkerrors` pkg? so that they are imported as `sdkerrors.<...>` instead of `sdk.<...>`

Reply:
no, they're typed errors for decimals. They're specific errors defined near the scope of the actual type(s).